### ISSUE TITLE
 ERROR Nuxt module should be a function: [object Object]

### ISSUE THREAD

**Issue by luc122c** [harlan-zw/nuxt-seo#360]
### 🐛 The bug

I'm getting the following error in my projects and on StackBlitz when upgrading to 2.0.0.

```
ERROR  Nuxt module should be a function: [object Object]

  at loadNuxtModuleInstance (node_modules/@nuxt/kit/dist/index.mjs:2555:11)
  at async Module.installModule (node_modules/@nuxt/kit/dist/index.mjs:2478:47)
  at async initNuxt (node_modules/nuxt/dist/index.mjs:4701:5)
  at async loadNuxt (node_modules/nuxt/dist/index.mjs:4871:5)
  at async loadNuxt (node_modules/@nuxt/kit/dist/index.mjs:2722:19)
  at async Object.run (node_modules/nuxi/dist/chunks/prepare.mjs:76:18)
  at async runCommand$1 (node_modules/nuxi/dist/shared/nuxi.bd0a2fa0.mjs:1680:16)
  at async Module.runCommand (node_modules/nuxi/dist/shared/nuxi.bd0a2fa0.mjs:2051:10)
  at async Object.setup (node_modules/nuxi/dist/chunks/add2.mjs:214:5)
  at async runCommand$1 (node_modules/nuxi/dist/shared/nuxi.bd0a2fa0.mjs:1652:5) 



[18:31:25]  ERROR  Nuxt module should be a function: [object Object]

```

### 🛠️ To reproduce

https://stackblitz.com/edit/nuxt-seo-2

### 🌈 Expected behavior

No error. 

### ℹ️ Additional context

I was running RC 23 and didn't have this issue.

**Comment by nosizejosh** [comment]
just found this package and first install gave same error.

**Comment by kilakewe** [comment]
@harlan-zw it may be related to how the build module was configured. In the compiled, I'm seeing references to your computer.

For example in: node_modules/@nuxtjs/seo/dist/const.mjs

```
import jiti from "file:///home/harlan/packages/nuxt-seo-solo/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/lib/index.js";

/** @type {import("/home/harlan/packages/nuxt-seo-solo/src/const")} */
const _module = jiti(null, {
  "esmResolve": true,
  "interopDefault": true,
  "alias": {
    "@nuxtjs/seo": "/home/harlan/packages/nuxt-seo-solo"
  }
})("/home/harlan/packages/nuxt-seo-solo/src/const.ts");
```

**Comment by StrangeRanger** [comment]
I unfortunatley got the same problem. I've tried configuring a resh project and installing it, just in case it was other configurations, but it still happens. I found that it happens **ONLY** when you place `@nuxtjs/seo` in the modules property within `nuxt.config.ts`, such as below:

```ts
// https://nuxt.com/docs/api/configuration/nuxt-config
export default defineNuxtConfig({
  compatibilityDate: '2024-11-01',
  devtools: { enabled: true },
  modules: [
    "@nuxtjs/seo"
  ]
})
```

**Comment by victor-ponamariov** [comment]
Same here =\ checked older issues, suggesting installing color-mode, which didn't help.
and it happens only if you add `@nuxtjs/seo` in modules.

**Comment by BayBreezy** [comment]
Same here. Removing the module from the list of modules got rid of the error.

**Comment by harlan-zw** [comment]
Thank you all, sorry for the mishap, 3am releases :laughing: . Looks like the module was getting stubbed before the `dist` files were being pushed up. It's fixed in 2.0.2.
